fortuna/src/chain/reader.rs

```diff
@@ -1,11 +1,38 @@
 use {
     anyhow::Result,
     axum::async_trait,
-    ethers::types::Address,
+    ethers::types::{
+        Address,
+        BlockNumber as EthersBlockNumber,
+    },
 };
 
 pub type BlockNumber = u64;
 
+/// A block status.
```

Comment:
This is not helping 😅  you can remove the comment or write a more descriptive one